client/src/pages/Browse.jsx

```diff
@@ -115,11 +118,19 @@ const Browse = () => {
               )}
             </div>
 
+            {/* browse watch list */}
+            <ContentRow
+              heading={"Watch List"}
```

Comment:
{} are unnecessary when passing a string value

Reply:
let it be bro looks consistent. What you say?

Reply:
Sure, anyhow there is no harm in any of it.